webapp/src/components/header/settings-dialog/SettingSection.tsx

```diff
@@ -56,7 +57,9 @@ export const SettingSection: React.FC<ISettingsSectionProps> = ({ setting, conte
                             <Switch
                                 label={feature.label}
                                 checked={feature.enabled}
-                                disabled={feature.inactive}
+                                disabled={
+                                    !!feature.inactive || (key === FeatureKeys.MultiUserChat && !AuthHelper.isAuthAAD())
```

Comment:
!!feature.inactive???  Isn't that the same as feature.inactive, or does it mean something special in JavaScript that I don't know about?

Reply:
in this case it should be, it's ESLint that is confused here:
<img width="1270" alt="image" src="https://github.com/microsoft/chat-copilot/assets/52973358/fdee316b-a090-4cd9-9085-fdd90cdd39af">

it is suggesting a nullish coalescing operator, I'm guessing because it sees that the variable is `boolean | undefined`, but that will incorrectly change the logic when `feature.inactive` is `false`, since the 2nd condition won't be evaluated.
 adding a `!!` forces the type to be just `boolean`, allowing ESLint to play nicely.